static/index.html

```diff
@@ -4,55 +4,67 @@
+			<h1>👻  <u>L</u>ightning PayJ<u>oin</u>  👻</h1>
+			<p style="color: orange; text-align: center;">🎃 Halloween Alpha [experimental] | Avoid sp👀ks</p>
+			<h2>Queue batches of lightning channels to open in a single transaction</h2>
 		</header>
 		<main lass="center-axyz">
 			<form action="/pj/schedule" method="post" enctype="application/x-www-form-urlencoded"
 				style="max-width: 750px; margin:0 auto; padding:2rem" x-flex direction="column">
-				<p style="color: orange">Halloween-quality pre-alpha software. A lil spooky!</p>
-				<div>
-					<label for="reserve">
-						<details>
-							<summary>
-								Anchor Reserve Deposit (sats)</label>
-							</summary>
-							New nodes need a reserve output of at least 10,000 sats. Set this to 10,000 fo a new node. 0 is fine thereafter.
-						</details>
-					</label>
-					<input type="number" name="wallet_amount" id="wallet" list="defaultReserveSats" min="0" step="1">
-					<datalist id="defaultReserveSats">
-						<option value="10000"></option>
-					</datalist>
-				</div>
-				<div>
-					<label for="fee_rate">Maximum fee rate (sats/vB)</label>
-					<input type="text" name="fee_rate" id="feerate" value="1">
-				</div>
 				<fieldset>
-					<legend>Channels to Open</legend>
+					<legend>Config</legend>
+					<div>
+						<label for="reserve">
+							<details>
+								<summary>
+									Anchor Reserve Deposit (sats)
+								</summary>
+								New nodes need a reserve output of at least 10,000 sats. Set this to 10,000 fo a new node. 0 is fine thereafter.
+							</details>
+						</label>
+						<input type="number" name="wallet_amount" id="wallet" list="defaultReserveSats" min="0" step="1" value="0">
+						<datalist id="defaultReserveSats">
+							<option value="10000"></option>
+						</datalist>
+					</div>
+					<div>
+						<label for="fee_rate">Maximum fee rate (sats/vB)</label>
+						<input type="text" name="fee_rate" id="feerate" value="1">
+					</div>
+				</fieldset>
+				<fieldset>
+					<legend>Queue Channels to Open</legend>
 					<!-- channels -->
 					<x-grid columns=2 id="channels">
 						<label id="destinationLabel" span=1>Destination Node</label><label id="capacityLabel" span=1>Channel Capacity (sats)</label>
-						<input type="text" name="channels[0][node]" aria-labelledby="destinationLabel" span=1><input type="number" name="channels[0][amount]" min="20000" step="1" aria-labelledby="capacityLabel" span=1><!-- LND enforces minimum 20k sats channel -->
-						<button type="button" onclick="add_channel()" order=1>Add channel</button>
+						<input type="text" id="input_node" name="channels[0][node]" aria-labelledby="destinationLabel" placeholder="node@host:port" span=1 required><input type="number" id="input_sats" name="channels[0][amount]" min="20000" step="1" aria-labelledby="capacityLabel" span=1 required><!-- LND enforces minimum 20k sats channel -->
```

Comment:
Yep fixes maintaining UI state. Great spot because could not recall what wtf i was doing here but now i see:

The `add_channel()` function takes the values from this `elementId` to use to create the table
```
table.innerHTML = table.innerHTML + `<input type="text" value="${input_node_uri}" name="channels[${++channels}][node]" aria-labelledby="destinationLabel" span="1"><input type="number" value="${input_sats}" name="channels[${channels}][amount]" min="20000" step="1" aria-labelledby="capacityLabel" span="1">`;
```
The `<input>s` we are adding to the table have no `id` so no duplicates.

Should we swap to appendChild now?